src/wombat.js

```diff
@@ -5623,8 +5623,8 @@ Wombat.prototype.initTopFrame = function($wbwindow) {
         false
       );
     }
-    $wbwindow.__WB_orig_parent = $wbwindow.parent;
-    $wbwindow.parent = replay_top;
+    //$wbwindow.__WB_orig_parent = $wbwindow.parent;
```

Comment:
nit: if these lines no longer are needed they should be removed not commented out